openrazer.spec

```diff
@@ -152,10 +152,22 @@ dkms remove -m %{dkms_name} -v %{dkms_version} --rpm_safe_upgrade --all
 %else
 
 %posttrans -n openrazer-kernel-modules-dkms
-#!/bin/sh
+#!/bin/bash
 set -e
 
-dkms install %{dkms_name}/%{dkms_version}
+get_kernels() {
+  if command -v rpm &> /dev/null; then
+    rpm -qa kernel | sort -uV | sed 's/kernel-//'
+  else
+    ls /lib/modules | sort -uV  
+  fi
+}
+
+kernels=$(get_kernels)
+
+for version in $kernels; do
```

Comment:
How about directly doing `for version in $(get_kernels); do` ? We'd save two lines that way